android/kotlin/v2/AmpliApp/app/src/main/java/com/amplitude/ampli/Ampli.kt

```diff
@@ -508,6 +513,7 @@ open class Ampli {
         if (this.client?.configuration?.plan == null) {
             this.client?.configuration?.plan = defaultObservePlan
         }
+        this.client?.add(IngestionMetadataErichmentPlugin())
```

Comment:
Ampli should set in the sourceName and version to the Plugin.
```suggestion
        this.client?.add(IngestionMetadataErichmentPlugin(IngestionMetaData("android-kotlin-ampli", "1.0.0")))
```